src/NuGet.Clients/NuGet.Credentials/Resources.resx

```diff
@@ -155,6 +155,6 @@
     <comment>{0} - plugin path</comment>
   </data>
   <data name="SecurePluginWarning_UseInteractiveOption" xml:space="preserve">
-    <value>The plugin credential provider could not acquire credentials. Authentication may require manual action. Consider re-running the command with --interactive or /p:NuGetInteractive="true"</value>
+    <value>The plugin credential provider could not acquire credentials. Authentication may require manual action. Consider re-running the command with --interactive in dotnet.exe, /p:NuGetInteractive="true" in MSBuild or removing the -NonInteractive switch in NuGet.exe</value>
```

Comment:
For a customer will `dotnet.exe` and `Nuget.exe`? Consider:

`...command with --interactive for 'dotnet', /p:NuGetInteractive="true" in MSBuild or removing the -NonInteractive switch for 'NuGet'

My preference for for instead of in is small. My concern is about including '.exe' when the user never seems that, no matter how correct you are.

Reply:
I think that sounds good. 
Fixed it per the suggestion.